Source/WebKit/WebProcess/WebPage/CoordinatedGraphics/LayerTreeHost.cpp

```diff
@@ -369,8 +369,12 @@ void LayerTreeHost::didFlushRootLayer(const FloatRect& visibleContentRect)
 
 void LayerTreeHost::commitSceneState(const RefPtr<Nicosia::Scene>& state)
 {
+    WTFBeginSignpost(this, LayerTreeHostCommitScene);
+
     m_isWaitingForRenderer = true;
     m_compositor->updateSceneState(state, ++m_compositionRequestID);
+
+    WTFEndSignpost(this, LayerTreeHostCommitScene);
```

Comment:
I'm not sure it's useful to know the duration of updateSceneState, it's just adding an element to a Vector and scheduling a run loop source. It should always be fast.

Reply:
Removed